### DISCUSSION TITLE
DEV_SSR flag disabling

### DISCUSSION THREAD

**Discussion by TalhaAhmedAI** [gatsbyjs/gatsby#29308] [Help] [answered]
Hi,

Whenever I run `Gatsby develop` I'm forcefully entered into this testing which I don't want to be a part of, this testing further slows down the development process which is very frustrating. Anyone here can suggest how to opt out from being a test subject.

```
info We're shipping new features! For final testing, we're rolling them out first to a small % of Gatsby users and your site was automatically chosen as one of them. With your help, we'll then release them to everyone in the next minor release.  We greatly appreciate your help testing the change. Please report any feedback good or bad in the umbrella issue. If you do encounter problems, please disable the flag by setting it to false in your gatsby-config.js like:  flags: {   THE_FLAG: false }  The following flags were automatically enabled on your site: - DEV_SSR · (Umbrella Issue) · Server Side Render (SSR) pages on full reloads
```

Thanks

**Comment by LekoArts** [comment, marked as answer]
Hi!

The message itself already gives you the answer:

> If you do encounter problems, please disable the flag by setting it to false in your gatsby-config.js like:  flags: {   THE_FLAG: false }

So if you set `DEV_SSR: false` then you'll definitely be opted-out. Please provide your feedback about "slows down the development process" in https://github.com/gatsbyjs/gatsby/discussions/28138 - Thanks!
